osu.Game/Overlays/Direct/DownloadButton.cs

```diff
@@ -76,6 +76,9 @@ private void load(OsuColour colours, OsuGame game, BeatmapManager beatmaps)
         {
             this.colours = colours;
 
+            beatmaps.ItemAdded += (set, existing, silent) => updateState(BeatmapSet.Value);
```

Comment:
why is this added? this logic is already in the underlying class, no?

Reply:
Sorry, I overlooked that :(